services/notification/src/app.py

```diff
@@ -170,9 +176,8 @@ def effects_reactor_callback(world):
 def handler(event, context, **kwargs):
     status = cfnresponse.SUCCESS
     world = {}
-    logger.info('WTF')
     try:
-        logger.info(f'Processing event {event}')
+        logger.info(f'Processing event {json.dumps(event)}')
```

Comment:
what happens if event isn't JSON, also is there anything sensitive in the event that we might not want to log?

Reply:
It will be `json.dump`able. Nothing sensitive, it's all data we've discovered about the customer's account.